tools/lib/bpf/libbpf.h

```diff
@@ -104,8 +104,15 @@ struct bpf_object_open_opts {
 	 * struct_ops, etc) will need actual kernel BTF at /sys/kernel/btf/vmlinux.
 	 */
 	const char *btf_custom_path;
+	/* Pointer to the custom BTF object to be used for BPF CO-RE relocations.
+	 * This custom BTF completely replaces the use of vmlinux BTF
+	 * for the purpose of CO-RE relocations.
+	 * NOTE: any other BPF feature (e.g., fentry/fexit programs,
+	 * struct_ops, etc) will need actual kernel BTF at /sys/kernel/btf/vmlinux.
+	 */
```

Comment:
this comment is the exact same comment as the one for btf_custom_path (which makes sense). Couldn't we use 1 comment for both (by changing the first one instead of duplicating it).